src/util/util.js

```diff
@@ -1,3 +1,13 @@
+export function compare(a, b) {
+  if (a.value < b.value) {
+    return 1
+  }
+  if (a.value > b.value) {
+    return -1
+  }
+  return 0
+}
```

Comment:
Tu poderia simplificar esse código da seguinte forma:
```suggestion
export function compare(a, b) {
  return b.value - a.value
}
```